### PR TITLE
New sample: Geotriggers

### PR DESCRIPTION
**Pull request by ZackAllen**

### CODE REVIEW

File: src/WPF/ArcGISRuntime.WPF.Viewer/Samples/Location/LocationDrivenGeotriggers/LocationDrivenGeotriggers.xaml.cs [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+        private async Task Initialize()
+        {
+            try
+            {
+                // This sample uses a web map with a predefined tile basemap, feature styles, and labels.
+                MyMapView.Map = new Map(new Uri("https://www.arcgis.com/home/item.html?id=6ab0e91dc39e478cae4f408e1a36a308"));
+                await MyMapView.Map.LoadAsync();
+
+                // Get service feature tables from the map to create GeotriggerMonitors for.
+                _gardenSections = ((FeatureLayer)MyMapView.Map.OperationalLayers.ToArray()[0]).FeatureTable as ServiceFeatureTable;
```
Comment (nCastle1) [inline src/WPF/ArcGISRuntime.WPF.Viewer/Samples/Location/LocationDrivenGeotriggers/LocationDrivenGeotriggers.xaml.cs:66]: `ToArray` step isn't necessary

**Review by nCastle1 — CHANGES_REQUESTED** [review]
Sample looks great, just a few suggestions

File: src/WPF/ArcGISRuntime.WPF.Viewer/Samples/Location/LocationDrivenGeotriggers/LocationDrivenGeotriggers.xaml.cs [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+                                _features.Add(geotriggerFeature);
+                            }
+                            catch (Exception ex)
+                            {
+                                MessageBox.Show(ex.Message, ex.GetType().Name, MessageBoxButton.OK, MessageBoxImage.Error);
+                            }
+                        }
+
+                        // Add the feature to the list view.
+                        _displayedFeatures.Add(_features.First(f => f.Name == fenceInfo.Message));
```
[the remark this answers is not in the document]
Comment (ZackAllen) [reply src/WPF/ArcGISRuntime.WPF.Viewer/Samples/Location/LocationDrivenGeotriggers/LocationDrivenGeotriggers.xaml.cs:174]: With the instructions, I'll pass on this.

File: src/Android/Xamarin.Android/Samples/Location/LocationDrivenGeotriggers/LocationDrivenGeotriggers.cs [hunk 3]
```diff
@@ -0,0 +1,298 @@
...
+                // Update the UI with the current section and points of interest.
+                _sectionButton.Text = $"Section: {_currentSection}";
+
+                if (_pointsOfInterest.Any())
+                {
+                    _poiLabel.Text = $"Points of interest: {string.Join(", ", _pointsOfInterest)}";
+                }
+                else
+                {
+                    _poiLabel.Text = $"Points of interest:";
```
Comment (nCastle1) [inline src/Android/Xamarin.Android/Samples/Location/LocationDrivenGeotriggers/LocationDrivenGeotriggers.cs:217]: `$` not needed

**Review by nCastle1 — APPROVED** [review]

**Pull request merged**